pkg/app/web/src/modules/deployments/index.ts

```diff
@@ -177,6 +182,14 @@ export const cancelDeployment = createAsyncThunk<
   }
 );
 
+export const updateMinUpdatedAt = createAsyncThunk<
+  void,
+  void,
+  { state: AppState }
+>("deployments/updateMinUpdate", async (_) => {
+  return;
+});
```

Comment:
I just try to update minUpdatedAt in the state. Is it proper way to make empty function and set extraReducer at L261?

Reply:
I can say that we can think from a different point of view like: do we need this `minUpdatedAt` stored in this deployment storage? Since I guess we can calculate it from the last deployment element of the store. Or even we keep this `minUpdatedAt` stored in the state, we can still not trigger to calculate it in `pkg/app/web/src/components/deployments-page/index.tsx:L138` but calculate it once we fetched deployments. What do you think?

Reply:
For sure. It might be enough to store in the state in the `DeploymentIndexPage` component, it's worth considering.

Reply:
In the end, I settled on storing `minUpdatedAt` in the deployment storage to simplify the logic.
As you mentioned, the dispatch to update minUpdatedAt is unneeded, hence I removed it.